pydicom/overlay_data_handlers/__init__.py

```diff
@@ -0,0 +1,8 @@
+
+import sys
+
+import pydicom.overlays.numpy_handler as _np_handler
+
+globals()['numpy_handler'] = _np_handler
+# Add to cache - needed for pytest
+sys.modules['pydicom.overlay_data_handlers.numpy_handler'] = _np_handler
```

Comment:
Can't this be done in the tests, if needed there? And why is it needed?

Reply:
Whoops, that was leftover from when I was halfway through switching to the new imports. Removed.